test/flint/fmpz_mod-test.jl

```diff
@@ -128,6 +131,21 @@ end
   @test_throws Exception R6(R22(1))
   @test_throws Exception R2(R3(1))
   @test_throws Exception R3(R2(1))
+
+  ZZmod720,_ = residue_ring(ZZ,720)
```

Comment:
This is a different ring, so the test for that one should be in `nmod-test.jl`.

We also generally place a space after commas in our code